### PR TITLE
FEATURE comments backend

### PR DESCRIPTION
**Pull request by moelheni**
REF [16](https://trello.com/c/k4grsH9d)

### CODE REVIEW

**Comment by moelheni** [comment]
@SamiDghim @thameursaadi @DoniaMroua verify this please

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+class CommentsController < ApplicationController
+  before_action :set_comment, only: [:show, :edit, :update, :destroy]
+
```
Comment (moelheni) [inline app/controllers/comments_controller.rb:3]: User needs to be signed in to do create a comment

**Comment by moelheni** [comment]
Do I merge this ?
ping @SamiDghim  @thameursaadi  @achr3f

**Comment by medBelaid** [comment]
Done

**Pull request merged**